test_setup/cy_interface/interface.json

```diff
@@ -482,7 +487,7 @@
                                              "[data-cy=\"connectObject options 16c2d1b8-4bbe-4f8d-b80e-5dc9ab61846c bd49d7b7-697b-4f0f-a4f4-12ca407fdc65 8129e697-a2a3-4fd1-a576-ec758cd54662\"]"
                                           ],
                                           "categories": [
-                                             "[data-cy=\"connectObject options 43ab8092-5878-4cf4-a98f-6e8798ff2117 84157ae7-caf3-4eae-bdf4-bf374b837d3c f807c3b6-1359-4cb6-823f-ccc1bf676e97\"]"
+                                             "[data-cy=\"connectObject options 43ab8092-5878-4cf4-a98f-6e8798ff2117 84157ae7-caf3-4eae-bdf4-bf374b837d3c 8129e697-a2a3-4fd1-a576-ec758cd54662\"]"
```

Comment:
If this isn't passing, you might want to make it search only by the first part: remove that last uuid, and add a `^`
like:
```
"[data-cy^=\"connectObject options 43ab8092-5878-4cf4-a98f-6e8798ff2117 84157ae7-caf3-4eae-bdf4-bf374b837d3c\"]"
```

Reply:
Oh, thank you Isaac